src/utils/vc.js

```diff
@@ -170,11 +237,13 @@ export async function verifyCredential(credential, {
   });
 
   // Check for revocation only if the credential is verified and revocation check is needed.
-  if (credVer.verified && isRevocationCheckNeeded(credential.credentialStatus, forceRevocationCheck, revocationApi)) {
-    const revResult = await checkRevocationStatus(credential, revocationApi);
-    // If revocation check fails, return the error else return the result of credential verification to avoid data loss.
-    if (!revResult.verified) {
-      return revResult;
+  if (credVer.verified) {
```

Comment:
This is functionally equivalent to what was before, why was the nesting introduced in place of the and operator?

Reply:
i was, before the final updates, doing something else here. dont think nesting is required here anymore